app/views/spree/checkout/payment/_bolt.html.erb

```diff
@@ -0,0 +1,15 @@
+<div id="bolt-container"
+  data-publishable-key="<%= payment_method.publishable_key %>"
+  data-payment-method-id="<%= payment_method.id %>"
+></div>
+<div id="payment-status-container" style="display: none;">
+  <button id="bolt-submit-button" type="button">Submit</button>
+</div>
+<button id="bolt-card-button" type="button">Pay</button>
+
+<script
```

Comment:
What do you think to use the `content_for` to load it in the footer or head section?

Reply:
I'd prefer loading it in the body rather than the head for efficiency reasons, and in that case I'm not sure how using `content_for :footer` would be better than the current implementation

Reply:
What are the `efficiency reasons`? I think that content `content_for` will have the same efficiencies

Reply:
This is also required for SSO. ref https://github.com/nebulab/solidus_bolt/pull/53/commits/b4a29a49c378f0d491babba469511c3005ee3730